zerver/openapi/zulip.yaml

```diff
@@ -3889,9 +3889,8 @@ paths:
                               description: |
                                 Event sent to administrators when the [data export
                                 consent][help-export-consent] status for a user changes, whether due
-                                to a user changing their consent preferences or a user being created
-                                or reactivated (since user creation/activation events do not contain
-                                these data).
+                                to a user changing their consent preferences or a user being reactivated
+                                (since user activation events do not contain these data).
```

Comment:
I don't think of it as a guarantee of the API that user creation events will not be sent -- I expect we will send them if it is true during user creation, which I expect future changes will make possible. So I wanted to write the API docs in a way that would correctly describe the intended behavior in that (currently impossible) situation. It doesn't really matter since client implementations don't care whether it was a created or reactivated user, but that was why I wrote it this way.